curriculum/challenges/english/15-javascript-algorithms-and-data-structures-22/learn-fetch-and-promises-by-building-an-fcc-authors-page/63ff70d3ee568219332868f3.md

```diff
@@ -0,0 +1,134 @@
+---
+id: 63ff70d3ee568219332868f3
+title: Step 5
+challengeType: 0
+dashedName: step-5
+---
+
+# --description--
+
+In order to start working with the data, you will need to use another `.then()` method.
+
+Chain another `.then()` to the existing `.then()` syntax. This time, what you need to pass into the `.then()` as the parameter is `data`. For the callback, use a curly brace because you will have more than one expression in the arrow function. For the `data`, log it to the console, for now, and open the console tto see what it looks like.
```

Comment:
```suggestion
In order to start working with the data, you will need to use another `.then()` method.

Chain another `.then()` to the existing `.then()` method. This time, pass in `data` as the parameter for the callback function. For the callback, use a curly brace because you will have more than one expression. Within your callback function, log `data` to the console to see what it looks like.
```